types/wordpress__components/color-picker/index.d.ts

```diff
@@ -1,19 +1,42 @@
 import { ComponentType } from 'react';
+import * as tinycolor from 'tinycolor2';
 
 declare namespace ColorPicker {
+    interface OnChangeCompleteValue {
+        color: tinycolor.Instance;
+        hex: string;
+        hsl: string;
+        hsv: string;
+        rgb: string;
+        oldHue: string;
+        source: 'rgb' | undefined;
+        draftHex: string;
+        draftHsl: string;
+        draftRgb: string;
+    }
```

Comment:
While I'm fairly confident this is _technically_ correct, I'm not totally sure if this expansion of this type is necessary. Any thoughts from other maintainers?